react/components/ExtensionPoint.tsx

```diff
@@ -191,22 +191,16 @@ const ExtensionPoint: FC<Props> = props => {
     </ExtensionPointComponent>
   )
 
-  if (runtime.preview && isRootTreePath) {
-    return (
-      <Fragment>
-        <LoadingBar />
-        {extensionPointComponent}
-      </Fragment>
-    )
-  }
-
   // "client" component assets are sent to server side rendering,
   // but they should display a loading animation.
   //
   // "lazy" components might never be used, so they don't necessarily
   // need a loading animation.
   return renderStrategy === 'client' && !runtime.amp ? (
-    <NoSSR onSSR={<Loading />}>{extensionPointComponent}</NoSSR>
+    <NoSSR onSSR={<Loading />}>
+      {runtime.preview && isRootTreePath && <LoadingBar />}
```

Comment:
you are only inserting the bar if the block has `renderStrategy === client`. 

since the bar is only inserted on root blocks, and they basically never have that render strategy, it will likely never render.

it should either be inserted on the `extensionPointComponent` itself, or in a fragment wrapping this ternary